service/backend/pchain/backend.go

```diff
@@ -25,7 +24,8 @@ var (
 
 type Backend struct {
 	fac                    *crypto.FactorySECP256K1R
-	networkIdentifier      *types.NetworkIdentifier
+	networkID              *types.NetworkIdentifier
+	networkHRP             string
```

Comment:
`networkHRP` is one of those quantities depending only on networkID. Prefetching it upon Backend construction